shuup/admin/modules/settings/views/__init__.py

```diff
@@ -0,0 +1,49 @@
+import six
+from django.contrib import messages
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.utils.translation import ugettext_lazy as _
+from django.views.generic import FormView
+
+from shuup.admin.modules.settings import ViewSettings
+from shuup.admin.modules.settings.forms import ColumnSettingsForm
+from shuup.admin.toolbar import get_default_edit_toolbar
+
+
+class ListSettingsView(FormView):
+    form_class = ColumnSettingsForm
+    template_name = "shuup/admin/edit_settings.jinja"
+
+    def dispatch(self, request, *args, **kwargs):
+        model = request.GET.get("model")
+        module = request.GET.get("module")
+        mod = __import__(module, fromlist=[model])
```

Comment:
This is not working in Python 2.7. TypeError at /sa/list-settings/ Item in ``from list'' not a string.